test/extended/deployments/deployments.go

```diff
@@ -290,8 +290,14 @@ var _ = g.Describe("[sig-apps][Feature:DeploymentConfig] deploymentconfigs", fun
 		})
 
 		g.It("resolve the image pull spec", func() {
+			isMicroShift, err := exutil.IsMicroShiftDeployment(oc)
```

Comment:
same question about the list of skipped tests and why the apigroup isn't appropriate.